### DISCUSSION TITLE
whistleblowing_identity fields

### DISCUSSION THREAD

**Discussion by apolastriMki** [globaleaks/globaleaks-whistleblowing-software#3646] [Q&A] [answered]
Abbiamo installato da circa un paio di anni la piattaforma senza riscontrare grossi problemi nella manutenzione ed aggiornamento, ora con le normative entrate in essere questo anno abbiamo necessità di rivedere la struttura dei form e riordinane gli step.Mi sono imbattuto, tramite il nostro avvocato, nella voce relativa al **Custodian**. Il funzionamenti mi è chiaro, l'ho testato sul docker in locale, e funziona  correttamente. Ma  il punto è : è possibile modificare i campi del template **whistleblowing_identity** ? Come è possibile cambiarne la struttura ed aggiungere ulteriori campi?

Antonio

**Comment by evilaliv3** [comment, marked as answer]
Thank you @apolastriMki 

Potete creare un vostro modello di domanda e una volta creato il modello ne trovate in calce un identificativo UUID4.

Per usare il vostro modello in alternativa alla nostra domanda whistleblower identity dovete poi aprire la domanda whistleblower identity ed inserire tale identificativo nel campo "Template override"

**Reply by apolastriMki** [reply]
Grazie mille per la celere risposta,
estendendo la tua risposta per futuro interesse da parte di altri utenti, riassumo la procedura.

1. si crea un modello di domanda ( questionari / questions -> modelli di domanda / question template) , questo è la domanda desiderata da secretare ( coi dati anagrafici per esempio)
2. Si crea la question / questionario di interessa nel tab Questionari
3. Al questionario di aggiunge lo step ed allo step la domanda **prefedefinta da template** , che è in essenza quella del template whistleblogin_identity.
4. A ques'utlima si sostituisce nel campo _template override_ l'UID presente nello step 1 al bottom del riquadro
5. Chiaramente è necessario salvare.

In tal modo il template creato viene sostituito al modello base secretato.

Già testato in locale e funziona correttamente!

Grazie mille
Antonio

**Comment by cienne1** [comment]
Ciao. Grazie per le indicazioni, ma non riusciamo a modificare il template identity per togliere la domanda a scelta multipla "Metodo di contatto alternativo", lasciando solo un campo di testo nel quale indicare l'email.
Puoi per favore inserire gli screen dei passaggi?

**Reply by apolastriMki** [reply]
Buongiorno, la questione è che non si modifica il modello del template identity, ma al suo posto si fa override con un template fatto da te ( che riporta un suo identificativo).In pratica si inserisce il template identity, ma all'interno dello stesso si inserisce l'UIDD del template che vorresti che fosse secretato
